Typo "Microsoft Teams Room"→"Microsoft Teams Rooms" (#8772)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-meeting-room-guidance.md

You can start with this plan by addressing all your non-AV or non-collaboration-equipped meeting spaces. You can still take advantage of Teams in your meeting rooms, even if those meeting rooms have not yet had Teams devices installed.

> *Audio:* A great bridge to a complete Teams Rooms is to plan for meeting technology by either providing your individual users with Teams-certified audio devices for smaller rooms and groups, or by deploying them in rooms ahead of a full Teams Rooms solution for larger spaces meant for larger groups where a small device won’t cut it.

Reviewer: ```suggestion
*Audio:* A great bridge to a complete Teams Room is to plan for meeting technology by either providing your individual users with Teams-certified audio devices for smaller rooms and groups, or by deploying them in rooms ahead of a full Teams Rooms solution for larger spaces meant for larger groups where a small device won’t cut it.
```

> ## What is an enhanced Microsoft Teams Room?

Reviewer: ```suggestion
## What is an enhanced Microsoft Teams room?
```

> In Microsoft’s hybrid workplace, the next generation of room standards are informed by the needs of both remote and in-room participants. An enhanced room is a Microsoft Teams Rooms designed to deliver an *elevated* hybrid meeting experience, fostering more natural and immersive connections while enabling everyone in the meeting to feel included, represented, and productive. The room follows a set of principles made real through a combination of Teams Rooms software and hardware capabilities and room configuration guidance.

Reviewer: ```suggestion
In Microsoft’s hybrid workplace, the next generation of room standards are informed by the needs of both remote and in-room participants. An enhanced room is a Microsoft Teams Room designed to deliver an *elevated* hybrid meeting experience, fostering more natural and immersive connections while enabling everyone in the meeting to feel included, represented, and productive. The room follows a set of principles made real through a combination of Teams Rooms software and hardware capabilities and room configuration guidance.
```

> The concept for the enhanced Microsoft Teams Rooms is designed around two key goals that are supported by a set of experience principles.

Reviewer: ```suggestion
The concept for the enhanced Microsoft Teams room is designed around two key goals that are supported by a set of experience principles.
```

-   What hardware upgrades and/or investments would need to be made.

> -   Most conference rooms are designed to maximize valuable square footage with a close relationship between room dimension and room capacity. An enhanced Microsoft Teams Rooms prioritizes the meeting experience above maximizing square footage. Not every organization can prioritize this way for every meeting space, as it could reduce the number of meeting rooms they can have.

Reviewer: ```suggestion
-   Most conference rooms are designed to maximize valuable square footage with a close relationship between room dimension and room capacity. An enhanced Microsoft Teams room prioritizes the meeting experience above maximizing square footage. Not every organization can prioritize this way for every meeting space, as it could reduce the number of meeting rooms they can have.
```

> Meeting content is surrounded by contextual fluid components like the agenda, tasks, and notes, which can be updated in real time, helping participants stay engaged and productive. Additionally, the meeting chat is brought clearly into view so that when joining a Teams meeting from a Teams Room, you can easily see and engage in the conversation in real time. In an enhanced Microsoft Teams Room, this content is displayed on a wide-aspect-ratio screen to maximize the size of meeting content and the number of participants that can be viewed simultaneously.

Reviewer: ```suggestion
Meeting content is surrounded by contextual fluid components like the agenda, tasks, and notes, which can be updated in real time, helping participants stay engaged and productive. Additionally, the meeting chat is brought clearly into view so that when joining a Teams meeting from a Teams room, you can easily see and engage in the conversation in real time. In an enhanced Microsoft Teams room, this content is displayed on a wide-aspect-ratio screen to maximize the size of meeting content and the number of participants that can be viewed simultaneously.
```

> This layout is supported across both single- and dual-display configurations. While the front row layout is especially impactful in an enhanced room, front row can be used in any Teams Rooms running on Windows, regardless of the furniture and hardware configuration.

Reviewer: ```suggestion
This layout is supported across both single- and dual-display configurations. While the front row layout is especially impactful in an enhanced room, front row can be used in any Teams room running on Windows, regardless of the furniture and hardware configuration.
```

#### Room capacity 

> This enhanced Microsoft Teams Rooms is optimized for between 6 - 8 people. 

Reviewer: ```suggestion
This enhanced Microsoft Teams room is optimized for between 6 and 8 people. 
```

## Recommended products for this space

> ### Microsoft Teams Rooms console and compute

Reviewer: ```suggestion
### Microsoft Teams Rooms console and computer
```
